### DISCUSSION TITLE
SAP connectors

### DISCUSSION THREAD

**Discussion by swathinaik94** [apache/camel-karavan#663] [Q&A] [answered]
1. Any sample connectors are available to connect SAP to send and receive IDOC?
2. If this is not available, will you have support for SAP connector for IDOC in future release?

**Comment by mgubaidullin** [comment, marked as answer]
1. Karavan uses Components from Apache Camel Framework
2. Apache Camel component for SAP IDOC may not be created as a part of Apache Camel Framework because of [SAP license restrictions](https://support.sap.com/en/product/connectors.html?anchorId=section_561121760)
3. You could create custom Kamelet based on [SAP Java IDoc library](https://support.sap.com/en/product/connectors/jco.html?anchorId=section_754631833) and use it in Karavan
